crates/polars-arrow/src/compute/cast/mod.rs

```diff
@@ -580,9 +580,23 @@ pub fn cast(
             LargeUtf8 => Ok(Box::new(utf8_to_large_utf8(
                 array.as_any().downcast_ref().unwrap(),
             ))),
-            Timestamp(TimeUnit::Nanosecond, None) => utf8_to_naive_timestamp_ns_dyn::<i32>(array),
```

Comment:
Is it possible to match on the time unit here and use that?

Reply:
I just changed it, thanks for the catch.